RecoEgamma/EgammaPhotonProducers/interface/ReducedEGProducer.h

```diff
@@ -90,18 +94,23 @@ class ReducedEGProducer : public edm::stream::EDProducer<> {
  const std::string outESRecHits_;
  const std::string outPhotonPfCandMap_;
  const std::string outGsfElectronPfCandMap_;
+ const std::string outGsfTracks_;  //add by me - Hien xinh dep
+ //const std::string outGsfTrackCores_; //add by me - Hien xinh dep
  const std::vector<std::string> outPhotonIds_;
  const std::vector<std::string> outGsfElectronIds_;
  const std::vector<std::string> outPhotonPFClusterIsos_;
  const std::vector<std::string> outGsfElectronPFClusterIsos_;
- 
+
  const StringCutObjectSelector<reco::Photon> keepPhotonSel_;
- const StringCutObjectSelector<reco::Photon> slimRelinkPhotonSel_; 
+ const StringCutObjectSelector<reco::Photon> slimRelinkPhotonSel_;
  const StringCutObjectSelector<reco::Photon> relinkPhotonSel_;
  const StringCutObjectSelector<reco::GsfElectron> keepGsfElectronSel_;
  const StringCutObjectSelector<reco::GsfElectron> slimRelinkGsfElectronSel_;
- const StringCutObjectSelector<reco::GsfElectron> relinkGsfElectronSel_; 
+ const StringCutObjectSelector<reco::GsfElectron> relinkGsfElectronSel_;
+
+ //add by Hien
+ // const StringCutObjectSelector<reco::GsfTrack> keepGsfTrackSel_;
+ //const StringCutObjectSelector<reco::GsfTrack> slimRelinkGsfTrackSel_;
+ //const StringCutObjectSelector<reco::GsfTrack> relinkGsfTrackSel_;
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant